static/app/views/settings/organizationIntegrations/integrationDetailedView.spec.tsx

```diff
@@ -216,15 +223,20 @@ describe('IntegrationDetailedView', function () {
       <IntegrationDetailedView
         {...TestStubs.routeComponentProps()}
         params={{integrationSlug: 'github'}}
-        organization={org}
+        organization={organization}
         location={TestStubs.location({query: {}})}
-      />
+      />,
+      {context: routerContext}
     );
 
     await userEvent.click(screen.getByText('features'));
 
     expect(
-      screen.getByRole('checkbox', {name: /Enable Pull Request Bot/})
+      screen.getByRole('checkbox', {name: /Enable Comments on Suspect Pull Requests/})
```

Comment:
Are there tests to show this button actually working? If not, could we add them?

Reply:
also if they do exist could ya link their location